contracts/RNGChainlinkV2.sol

```diff
@@ -0,0 +1,219 @@
+    require(_subscriptionId > 0, "RNGChainLink/subId-gt-zero");
+    require(_callbackGasLimit > 0, "RNGChainLink/gas-limit-gt-zero");
+    require(_requestConfirmations > 0, "RNGChainLink/requestConf-gt-zero");
+    require(_numWords > 0, "RNGChainLink/numWords-gt-zero");
+    require(_keyHash != bytes32(0), "RNGChainLink/keyHash-not-zero");
+
+    _vrfCoordinator = VRFCoordinatorV2Interface(vrfCoordinator_);
+
+    RequestConfig memory _sRequestConfig = RequestConfig({
+      subId: _subscriptionId,
+      callbackGasLimit: _callbackGasLimit,
+      requestConfirmations: _requestConfirmations,
+      numWords: _numWords,
+      keyHash: _keyHash
+    });
+
+    sRequestConfig = _sRequestConfig;
+
+    emit VrfCoordinatorSet(_vrfCoordinator);
+    emit RequestConfigSet(_sRequestConfig);
+  }
+
+  /* ============ External Functions ============ */
+
+  /// @inheritdoc RNGInterface
+  function requestRandomNumber()
+    external
+    override
+    onlyManager
+    returns (uint32 requestId, uint32 lockBlock)
+  {
+    RequestConfig memory _requestConfig = sRequestConfig;
+
+    uint256 _vrfRequestId = _vrfCoordinator.requestRandomWords(
+      _requestConfig.keyHash,
+      _requestConfig.subId,
+      _requestConfig.requestConfirmations,
+      _requestConfig.callbackGasLimit,
+      _requestConfig.numWords
+    );
+
+    sRequestId = _vrfRequestId;
+
+    requestCounter++;
+    uint32 _requestCounter = requestCounter;
+
+    requestId = _requestCounter;
+    chainlinkRequestIds[_vrfRequestId] = _requestCounter;
+
+    lockBlock = uint32(block.number);
+    requestLockBlock[_requestCounter] = lockBlock;
+
+    emit RandomNumberRequested(_requestCounter, msg.sender);
+  }
+
+  /// @inheritdoc RNGInterface
+  function isRequestComplete(uint32 _internalRequestId)
+    external
+    view
+    override
+    returns (bool isCompleted)
+  {
+    return randomNumbers[_internalRequestId] != 0;
+  }
+
+  /// @inheritdoc RNGInterface
+  function randomNumber(uint32 _internalRequestId)
+    external
+    view
+    override
+    returns (uint256 randomNum)
+  {
+    return randomNumbers[_internalRequestId];
+  }
+
+  /// @inheritdoc RNGInterface
+  function getLastRequestId() external view override returns (uint32 requestId) {
+    return requestCounter;
+  }
+
+  /// @inheritdoc RNGInterface
+  function getRequestFee() external pure override returns (address feeToken, uint256 requestFee) {
+    return (address(0), 0);
+  }
+
+  /// @inheritdoc RNGChainlinkV2Interface
+  function getSubscriptionId() external view override returns (uint64) {
+    return sRequestConfig.subId;
+  }
+
+  /// @inheritdoc RNGChainlinkV2Interface
+  function getVrfCoordinator() external view override returns (address) {
+    return address(_vrfCoordinator);
+  }
+
+  /// @inheritdoc RNGChainlinkV2Interface
+  function setKeyhash(bytes32 _keyhash) external override onlyOwner {
+    sRequestConfig.keyHash = _keyhash;
+
+    emit KeyHashSet(_keyhash);
+  }
```

Comment:
In addition to setting the keyhash, you'll want to add setters for each of the fields in RequestConfig.

Or you could just add a `setRequestConfig` function.  It's prob easiest, even if info gets repeated. You already have a `RequestConfigSet` event too.

Reply:
Done: https://github.com/pooltogether/pooltogether-rng-contracts/pull/22/commits/21e2671d259a0cdfa707810b125b7bf3c64cb7b5